### PR TITLE
feat/Landing: Events Banner

### PR DESCRIPTION
**Pull request by Eric-L-6**
Creates the events banner for the landing page.
Currently events divider is hardcoded, in future will need to be dynamic

image

### CODE REVIEW

File: src/routes/LandingPage/EventsBanner/EventsBanner.jsx
Comment (imagine-hussain) [inline src/routes/LandingPage/EventsBanner/EventsBanner.jsx:1]: `eslint-disable` is great for developing stuff (i usually have it on too) but we do need to remove this line before merging into main

File: src/routes/LandingPage/EventsBanner/index.scss
Comment (imagine-hussain) [inline src/routes/LandingPage/EventsBanner/index.scss:32]: I think perhaps this font-size is perhaps a little big; I think something thats approximately `2rem` would look great.

We've defined `1 rem` to be `18px` so that would come to ~`36px` but I think keeping the font-size in rem is a good idea.

Article on ems and rems if you're unfamiliar:
https://www.digitalocean.com/community/tutorials/css-rem-vs-em-units

File: src/routes/LandingPage/EventsBanner/EventsBanner.jsx
Comment (imagine-hussain) [inline src/routes/LandingPage/EventsBanner/EventsBanner.jsx:23]: Just something to consider for next sprint, would be to import `Typography` from `antd` and then use the `Text` object from that. You can have a look at the footerbar to see how it may be used and also their documentation:
https://ant.design/components/typography/

Don't worry about that before merging this in though, feel free to resolve thsi conversation as soon as you see it.

File: src/routes/LandingPage/EventsBanner/EventsBanner.jsx
Comment (imagine-hussain) [inline src/routes/LandingPage/EventsBanner/EventsBanner.jsx:30]: When EventsDivider is not used in `EventsBanner`, `rows` is not passed in.
In this case, `rows` defaults to `undefined` and then we do a comparison of it with `i` which starts as `0`.
`0 < undefined` defaults to falsey i believe so it won't crash but, it may be wise to add in a check for `rows` not being undefined

File: src/routes/LandingPage/EventsBanner/EventsBanner.jsx
Comment (imagine-hussain) [reply src/routes/LandingPage/EventsBanner/EventsBanner.jsx:23]: in an ideal world where this repo is in typescript, this would be less of an issue 😔

**Review by imagine-hussain — CHANGES_REQUESTED** [review]
Great work!
The highlight effect on the banner looks great. I'm happy to let you merge this once the following get addressed:
- remove the `eslint-disable` 
- make the `font-size` of `.evnts-single` to something smaller. I've recommended a size but you can have a play with it to what you think is good.
- 

## Notes for next sprint
These don't need to be addressed before merging this branch in but just good to keep in the back of ur mind as you start with your next card.

- Have a look at the `Typography` package from antd. You can see usage of it in the footerbar and Even's event preview section once he merges that in.
- Honestly, the hard-coding for this isnt actually the worst thing and we can get away with it partially just by allowing those excess `event`s in the banner to just overflow (as long as that's done in a manner that it silently overflows horizontally without affecting elements below)`
-

File: src/routes/LandingPage/EventsBanner/EventsBanner.jsx
Comment (Eric-L-6) [reply src/routes/LandingPage/EventsBanner/EventsBanner.jsx:1]: All good, will get it changed

File: src/routes/LandingPage/EventsBanner/EventsBanner.jsx
Comment (imagine-hussain) [reply src/routes/LandingPage/EventsBanner/EventsBanner.jsx:30]: Oh, my bad, I saw the props but totally missed the default values.

**Review by imagine-hussain — APPROVED** [review]
My bad, missed the default values.
I'd say, just for the sake of getting this merged, I manually moved the size to  `2rem` but, feel free to change this to anything you prefer in the next sprint's PR.

Great work, lets get this merged.

**Pull request merged**
